conda_build/jinja_context.py

```diff
@@ -391,11 +391,25 @@ def compiler(language, config, permit_undefined_jinja=False):
 
         # support cross compilers.  A cross-compiler package will have a name such as
         #    gcc_target
-        #    gcc_linux-cos5-64
+        #    gcc_linux-cos6-64
         compiler = '_'.join([compiler, config.variant['target_platform']])
     return compiler
 
 
+def cdt(package_name, config, permit_undefined_jinja=False):
+    """Support configuration of Core Dependency Trees."""
+    cdt_name = 'cos6'
+    arch = config.host_arch or config.arch
+    cdt_arch = 'x86_64' if arch == '64' else 'i686'
+    if config.variant:
+        cdt_name = config.variant.get('cdt_name', cdt_name)
+        cdt_arch = config.variant.get('cdt_arch', cdt_arch)
+    if ' ' in package_name:
+        return (package_name.split(' ')[0] + '-' + cdt_name + '-' + cdt_arch + ' ' + ' '.join(package_name.split(' ')[1:]))
```

Comment:
This is more of a code style suggestion than a requirement: please don't ever return from more than one place.  Store a value in one of many ways, and then have one return line that just returns that value.

It's a nitpicky thing (sorry) but I find that it makes code easier to follow.